content/rs/administering/creating-databases/_index.md

```diff
@@ -93,21 +93,23 @@ for this database. Minimum RAM portion is 10%, and maximum RAM portion is 50%.
         To protect against loss of data stored in RAM,
         you can enable data persistence and select to store a copy of the data on disk with snapshots or Append Only File.
 
-    - **Default database access** - When you configure a password for your database,
-        all connections to the database must authenticate with the [AUTH command](https://redis.io/commands/auth).
-        If you also configure an access control list, connections can specify other users for authentication,
-        and requests are allowed according to the Redis ACLs specified for that user.
+    - **Default database access** - To require a password for all database connections,
```

Comment:
I think this is inaccurate. 
Default database access is used to connect to the DB w/o username. It's mostly for backwards compatibility (and we can say that and encourage using users).
It is important to say that in order to require password for all connections you need to:
1 - enable default access WITH password
(or) 2 -  disable it and use username+pwd.

Reply:
@AlonMagrafta Please review again.

Reply:
@AlonMagrafta could you review this? Is this PR still relevant and accurate?